Kiosk/App/CardHandler.swift

```diff
@@ -1,9 +1,14 @@
 import UIKit
-import ReactiveCocoa
+import RxSwift
 
 class CardHandler: NSObject, CFTReaderDelegate {
 
-    let cardSwipedSignal = RACSubject()
+    private let _cardStatus = BehaviorSubject(value: "")
```

Comment:
Ah, I see. Seems good to me.

I'd like to standardize a pattern for representing mutable app state (whether that's external data model or UI state). perhaps this is it!